src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, 'Helvetica Neue', Arial, sans-serif;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
```

Comment:
Avoid fixing the size of containers like `.logo` unless it's a specific requirement. Fixed sizes can lead to issues with content overflow or unexpected layout behavior on different screen sizes or when the content changes.